README.md

```diff
@@ -85,7 +88,11 @@ philosophy, and have some tests.
 
 ## Any chance I can get a hosted and managed instance?
 
-Yup, stay tuned :) or get in touch with us.
+We've got you covered. [Astarte Cloud](https://astarte.cloud/cloud) is a complete hosted and managed
+solution to employ Astarte as a service, without having to worry about setting up your own
+infrastructure and its related costs. You can also choose specifically how many resources you need
+for your use-case and scale up your business model. Joining Astarte Cloud is as easy as [logging
```

Comment:
`scale up your business model` doesn't really make a lot of sense here. I'd simply remove it.